plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1893,6 +1893,20 @@ def get_limit_value(interaction_type, limit_type, cases):
 
         return limit_values
 
+    @staticmethod
+    def get_fundamental_frequency(species, n, v_a, Coulomb_log):
+        # This special case for computing the fundamental frequencies comes from page 33 of the NRL Formulary
```

Comment:
Make this a docstring 

Also add that that formula in the NRL formulary is a special case of the Maxwellian_avg_##_collision_freq (I think in the limit that T_a ~ T_b?)